content/rs/administering/database-operations/create-crdb.md

```diff
@@ -147,16 +141,12 @@ redis-cli is a simple command-line tool to interact with redis database.
     "123"
     ```
 
-1. Enter `exit` to exit the redis-cli context and enter `exit` again to exit the
-   RS container of node 1 in cluster 1.
-1. To see that the key replicated to cluster 2, repeat the steps to switch your
-   context into the RS container of node 1 in cluster 2, run the redis-cli and
+1. Enter the terminal of node 1 in cluster 2, run the redis-cli, and
```

Comment:
We don't mention Docker in the setup section so it does not belong here either. Also, this article is intended for production environments.